examples/gallery/plot/scatter.py

```diff
@@ -21,7 +21,7 @@
     projection="X10c/10c",
     frame=["xa0.2fg", "ya0.2fg", "WSrt"],
 )
-for color in ["blue", "orange", "green"]:
+for color in ["seagreen2", "chocolate1", "mediumpurple1"]:
```

Comment:
```suggestion
for color in ["seagreen2", "chocolate1", "mediumpurple1"]:
```
Not sure if these colors solve the problem (please correct me if I'm worong). Maybe it would be better to select colors from cpts that really focus on the color blindness issue.

Reply:
I agree, but colorblind friendly colorschemes are very limited, and transparency makes this case even worst, I've changed green to a grey, this change will produce greater differences on protanopia and deuteranopia, but purple will get closer to grey on tritanopia (the rarest of the three).

Everywhere i looked on how to choose colorblind friendly colors they say not to rely only color, but to create other ways to differentiate set, usualy changing marker. So i added this.

Also, i couldn't find colorblind friendly cpts on gmt.

Reply:
> transparency makes this case even worst,

Perhaps we can try to increase the transparency slightly?

Reply:
> 
> 
> > transparency makes this case even worst,
> 
> Perhaps we can try to increase the transparency slightly?

Agree, perhaps `transparency=50` gives a better result.

Reply:
No, @michaelgrund, "transparency makes this case even worst" means we sould **remove** transparency.
Like ```transparency=90``` or remove that line completely (which would be equal to ```transparency=100``` (default)).

Transparenncy makes it worst beacuse it blends the colors and changes their luminance when adding two or more symbols with equal **or** different colors, notting that color addition with colorblindness is a little bit tricky. That's why transparency should be avoided if we want it more coverage considering colorblindness.

Reply:
I think we're getting a bit confused here. `transparency=0` means opaque while `transparency=100` means invincible. We probably want to **reduce** transparency (i.e. increase opacity).

Reply:
Right @weiji14. I think we should not drop transparency here since it allows to deal with overplotting.